op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -264,9 +267,11 @@ func applyAllocsToState(db *state.StateDB, genesis *core.Genesis, config *params
 						log.Info("Account already exists with different code and is whitelisted, overwriting...", "address", k)
```

Comment:
Why do we have a whitelist that allows overwriting accounts?

Reply:
Because we know of one contract (`Create2Deployer`) which already exists but has different code on Alfajores.